platform/commonUI/edit/test/actions/SaveAsActionSpec.js

```diff
@@ -156,6 +156,28 @@ define(
                 expect(SaveAsAction.appliesTo(actionContext)).toBe(false);
             });
 
+            it("uses the editor capability to save the object", function () {
+                mockEditorCapability.save.andReturn(new Promise(function () {}));
```

Comment:
@BogdanAlexandru Let me get back to you re: `waitsFor`. re: CreationService, go ahead and remove it. It's a hangover from refactoring of the `SaveAction` and is obsolete.